apps/editing-toolkit/package.json

```diff
@@ -16,61 +16,10 @@
-		"newspack-carousel-block-editor": "calypso-build --env source='newspack-blocks/carousel-block-editor'",
-		"newspack-carousel-block-view": "calypso-build --env source='newspack-blocks/carousel-block-view'",
-		"newspack-blocks-carousel-block": "npm-run-all --parallel newspack-carousel-block-*",
-		"dev:newspack-blocks": "yarn run newspack-blocks",
-		"build:newspack-blocks": "NODE_ENV=production yarn run newspack-blocks",
-		"starter-page-templates": "calypso-build --env source='starter-page-templates'",
-		"dev:starter-page-templates": "yarn run starter-page-templates",
-		"build:starter-page-templates": "NODE_ENV=production yarn run starter-page-templates",
-		"event-countdown-block": "calypso-build --env source='event-countdown-block'",
-		"dev:event-countdown-block": "yarn run event-countdown-block",
-		"build:event-countdown-block": "NODE_ENV=production yarn run event-countdown-block",
-		"global-styles": "calypso-build --env source='global-styles'",
-		"build:global-styles": "NODE_ENV=production yarn run global-styles",
-		"dev:global-styles": "yarn run global-styles",
-		"jetpack-timeline": "calypso-build --env source='jetpack-timeline'",
-		"dev:jetpack-timeline": "yarn run jetpack-timeline",
-		"build:jetpack-timeline": "NODE_ENV=production yarn run jetpack-timeline",
-		"wpcom-block-editor-nux": "calypso-build --env source='wpcom-block-editor-nux'",
-		"dev:wpcom-block-editor-nux": "yarn run wpcom-block-editor-nux",
-		"build:wpcom-block-editor-nux": "NODE_ENV=production yarn run wpcom-block-editor-nux",
-		"wpcom-block-editor-welcome-tour": "calypso-build --env source='wpcom-block-editor-welcome-tour'",
-		"block-inserter-modifications": "npm-run-all --parallel block-inserter-modifications-*",
-		"block-inserter-modifications-contextual-tips": "calypso-build --env source='block-inserter-modifications/contextual-tips'",
-		"dev:block-inserter-modifications": "yarn run block-inserter-modifications",
-		"build:block-inserter-modifications": "NODE_ENV=production yarn run block-inserter-modifications",
-		"wpcom-block-editor-nav-sidebar": "calypso-build --env source='wpcom-block-editor-nav-sidebar'",
-		"dev:wpcom-block-editor-nav-sidebar": "yarn run wpcom-block-editor-nav-sidebar",
-		"build:wpcom-block-editor-nav-sidebar": "NODE_ENV=production yarn run wpcom-block-editor-nav-sidebar",
-		"block-patterns": "calypso-build --env source='block-patterns'",
-		"dev:block-patterns": "yarn run block-patterns",
-		"build:block-patterns": "NODE_ENV=production yarn run block-patterns",
-		"whats-new": "calypso-build --env source='whats-new'",
-		"dev:whats-new": "yarn run whats-new",
-		"build:whats-new": "NODE_ENV=production yarn run whats-new",
-		"dev": "node bin/npm-run-build.js --dev",
-		"build": "node bin/npm-run-build.js --build",
+		"dev": "node bin/npm-run-build.js",
+		"build": "NODE_ENV='production' node bin/npm-run-build.js",
+		"calypso-build": "calypso-build",
```

Comment:
I don't _think_ you need this. By default, `yarn <thing>` will look for `<thing>` in `node_modules/.bin` if it is not a defined `script`.

Reply:
I thought I tried that first, but I'll look at it again. I think it might not work if modules are installed to the calypso root and not to apps/editing-toolkit